### PR TITLE
[Feat] 여정 합류 

### PR DESCRIPTION
**Pull request by sumi-0011**
## 작업 내용
- 메인 여정 합류 PC ,mobile 디자인 구현

## 스크린샷
image

image

### CODE REVIEW

**Comment by depromeet-official** [comment]
#### 📝 Changed routes:


| [**/**](https://depromeet-8x941mjx6-depromeet-official-site.vercel.app/) |
|:-:|
|<a href="https://depromeet-8x941mjx6-depromeet-official-site.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/v2nZ4gqFpq007eqP3HJ46Sb-DwofyQR2mc5hjnvaMbc0-Z63clnn0VFfWKZKq_keSn6iApOItcGUWeQw8_Bgtw.png">(view full size)</a> |







Commit 5da8b92389e95f1131cae5aaead375deca1a3c57 (https://depromeet-8x941mjx6-depromeet-official-site.vercel.app).

File: src/components/TimerContainer/TimerContainer.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export function TimerContainer() {
   const time = useDiffDay(DEADLINE_DATE);
 
   return (
-    <div css={containerCss}>
+    <section css={containerCss}>
```
Comment (kimyouknow) [inline src/components/TimerContainer/TimerContainer.tsx:15]: 디테일 미쳤다...!

File: src/components/TimerContainer/TimerContainer.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ export function TimerContainer() {
...
       <div css={bgImageCss}>
```
Comment (kimyouknow) [inline src/components/TimerContainer/TimerContainer.tsx:16]: 디테일 미쳤다...!

**Review by kimyouknow — APPROVED** [review]
👍

**Pull request merged**